cross/fish/Makefile

```diff
@@ -0,0 +1,26 @@
+PKG_NAME = fish
+PKG_FULL_NAME = fish-shell
+PKG_EXT = tar.gz
+PKG_DOWNLOAD_METHOD = git
```

Comment:
I have to double check but I think the next ```2.2``` release is around the corner, maybe a few weeks.
 I will resubmit a pull request with a proper release when available.Thanks